internal/cryptogen/msp/msp.go

```diff
@@ -6,6 +6,9 @@ SPDX-License-Identifier: Apache-2.0
 package msp
```

Comment:
The MSP needs to be versioned like [here](https://github.com/hyperledger/fabric/blob/main/msp/mspimpl.go#L112-L139) from the reasons I outlined.